framework/src/base/MooseApp.C

```diff
@@ -1133,6 +1135,30 @@ void
 MooseApp::run()
 {
   TIME_SECTION(_run_timer);
+  if (isParamValid("show_docs") && getParam<bool>("show_docs"))
+  {
+    auto binname = appBinaryName();
+    if (binname == "")
+      mooseError("could not locate installed tests to run (unresolved binary/app name)");
+    auto docspath = MooseUtils::docsDir(binname);
+    if (docspath == "")
+      mooseError("no installed documentation found");
+
+    auto docmsgfile = MooseUtils::pathjoin(docspath, "docmsg.txt");
+    std::string docmsg = "file://" + MooseUtils::realpath(docspath) + "/index.html";
+    if (MooseUtils::pathExists(docmsgfile) && MooseUtils::checkFileReadable(docmsgfile))
+    {
+      std::ifstream ifs(docmsgfile);
+      std::string content((std::istreambuf_iterator<char>(ifs)),
+                          (std::istreambuf_iterator<char>()));
+      content.replace(content.find("$LOCAL_SITE_HOME"), content.length(), docmsg);
```

Comment:
Do we not want something like MOOSE_LOCAL_SITE_HOME? Or something that can be unique to application?

This probably isn't a big deal - just trying to think forward

Reply:
The URL is always already app-specific.